### PR TITLE
Fix style inconsistencies of "Verify this device" dialog

### PR DESCRIPTION
**Pull request by luixxiul**
Closes https://github.com/vector-im/element-web/issues/21572

This PR addresses style inconsistencies of the security key dialog and modal window, including

- Window's padding value (based on authentication modal window; see Image 1)
- Border-radius (see Image 1)
- Text color (based on Secure Key dialog; see Image 2)
- The warning sign of the footer message (see Image 2)
- Size of the close button (see image 2)

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

### Before

Current design
before2

Image 1
after1

Image 2
after

### After

after2

Perhaps the big red icon on the header should be replaced with something else.

after

type: defect

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix style inconsistencies of "Verify this device" dialog ([\#8167](https://github.com/matrix-org/matrix-react-sdk/pull/8167)). Fixes vector-im/element-web#21572. Contributed by @luixxiul.

Preview: https://pr8167--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by luixxiul** [comment]
It looks like the border-radius and the text color are also a little bit different. I would like to address them, including padding of the dialog, with a different PR.

**Comment by luixxiul** [comment]
No problem. Considering how much change this PR should include, I've replaced "fixes" with "addresses".

File: res/css/structures/auth/_SetupEncryptionBody.scss [hunk 1]
```diff
@@ -18,4 +18,5 @@ limitations under the License.
     @mixin ButtonResetWarning;
     color: $light-fg-color; // .mx_Dialog on _common.scss
     margin-top: $font-14px;
+    line-height: 1; // align the warning icon
```
Comment (luixxiul) [inline res/css/structures/auth/_SetupEncryptionBody.scss:21]: This change should fix this bug:

after (copy 1)

**Comment by luixxiul** [comment]
There is another dialog whose styles are also inconsistent due to the fact it uses `BaseDialog`.

before

Here the color of the warning icon is black.

**Comment by luixxiul** [comment]
Since it looks like it needs complicated changes of `SetupEncryptionDialog.tsx` more than changing some values of scss, I would like to leave it as it is for now.

**Review by turt2live — COMMENTED** [review]
I think this one needs design review before code review is possible.

**Comment by luixxiul** [comment]
Closing as stale.

**Pull request closed without merging**